src/godot_fmod.cpp

```diff
@@ -11,7 +11,6 @@ Mutex *Callbacks::mut;
 Fmod::Fmod() = default;
 
 Fmod::~Fmod() {
-    Fmod::shutdown();

```

Comment:
What if someone wants to use the library without the GDScript wrapper ?
Maybe we can have a check here to handle if already shutdown or not ?

Reply:
Well they have to shutdown FMOD themselves before imo.
Fmod is not initialized in the constructor of the class so I don't think it should be the job of the destructor to shutdown it.
If we still want to have a check, we have to move the logic from GDScript to C++.

Reply:
I'm fine with it. Maybe we can improve it later